README.rst

```diff
@@ -142,6 +142,16 @@ You should now be able to browse to a ``Genes`` page and search for
 ``CDC20`` or ``ENSG00000117399``.
 
 
+External URL Favicons
+^^^^^^^^^^^^^^^^^^^^^
+
+Mapr can automatically convert external URLs in mapr categories to clickable favicons.
+OMERO.web must be configured with the Django redis cache https://docs.openmicroscopy.org/omero/5/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.html?highlight=redis#configuring-omero-web which is used to cached the favicons that are obtained using a Google service.
```

Comment:
@will-moore I've updated this section. I can't find any official documentation on Google's `s2/favicons` API:
https://github.com/ome/omero-mapr/blob/89f9c87fba77be8f0e9b7a532400a0c6d57902a7/omero_mapr/mapr_settings.py#L41-L44